examples/gno.land/p/demo/forms/forms.gno

```diff
@@ -0,0 +1,131 @@
+package forms
+
+import (
+	"errors"
+	"std"
+	"time"
+
+	"gno.land/p/demo/seqid"
+)
+
+type Field struct {
+	Label string
+
+	/*
+		string: "string";
+		number: "number";
+		boolean: "boolean";
+		choice: "['Pizza', 'Schnitzel', 'Burger']";
+		multi-choice: "{'Pizza', 'Schnitzel', 'Burger'}";
+	*/
+	FieldType string
+
+	Required bool
+}
+
+type Form struct {
+	ID    string
+	Owner std.Address
+
+	Title       string
+	Description string
+	Fields      []Field
+
+	CreatedAt time.Time
+	openAt    *time.Time
+	closeAt   *time.Time
+}
+
+type Submission struct {
+	FormID string
+
+	Author std.Address
+
+	/* ["Alex", 21, true, 0, [0, 1]] */
+	Answers string // json
+
+	SubmittedAt time.Time
+}
+
+type FormDatabase struct {
```

Comment:
Nitpick: This could be `FormDB`

Reply:
done in https://github.com/gnolang/gno/pull/2604/commits/254c10fb363ca811410c0caea4e768b4e5ca959f